common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleIntegrationSuite.java

```diff
@@ -186,7 +186,12 @@ public void testFetchThreeSort() throws Exception {
 
   @Test
   public void testFetchInvalidShuffle() throws Exception {
-    registerExecutor("exec-1", dataContext0.createExecutorInfo("unknown sort manager"));
+    try {
```

Comment:
Yeah, this would be better as a separate test:

```
@Test(expect=TheExpectedException.class)
public void testInvalidShuffleManager() {
   registerExecutor("exec-1", dataContext0.createExecutorInfo("unknown sort manager"));
}
```

(Not sure about the names of the arguments, but it's something like that.)